libscanbuild/intercept.py

```diff
@@ -163,6 +163,15 @@ def write_exec_trace(filename, entry):
     :param filename:    path to the output execution trace file,
     :param entry:       the Execution object to append to that file. """
 
+    for param in entry.cmd[:]:
```

Comment:
Can this extracted to a separate method? Which would take the commands and would return a fixed command line. Would like to test it separately if we can.